overlord/servicestate/quota_handlers_test.go

```diff
@@ -200,6 +396,68 @@ func (s *quotaHandlersSuite) TestDoQuotaControlRemove(c *C) {
 	checkQuotaState(c, st, nil)
 }
 
+func (s *quotaHandlersSuite) TestDoQuotaControlRemoveRestartOK(c *C) {
```

Comment:
not really your fault since I'm quite guilty of not doing this myself, but looking at these tests they are very similar and could potentially be combined into a common test helper instead of duplicating the main body of executing the test